src/Symfony/Component/DependencyInjection/Compiler/RegisterAutoconfigureAttributesPass.php

```diff
@@ -42,7 +43,8 @@ public function accept(Definition $definition): bool
 
     public function processClass(ContainerBuilder $container, \ReflectionClass $class): void
     {
-        foreach ($class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF) as $attribute) {
+        $attributes = array_merge($class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF), $class->getAttributes(Lazy::class, \ReflectionAttribute::IS_INSTANCEOF));
```

Comment:
OK, second though: let's make them conflict :)
I was reviewing and considering that we now ignore the order of declaration of attributes when both are set.
We could preserve the order but that'd make the code more complex.
Let's add the same conflict rule that we have for Autowire and keep things simple.